packages/mantine-ui/src/SelectWidget/SelectWidget.tsx

```diff
@@ -0,0 +1,121 @@
+import { FocusEvent } from 'react';
+import {
+  ariaDescribedByIds,
+  enumOptionsIndexForValue,
+  enumOptionsValueForIndex,
+  labelValue,
+  EnumOptionsType,
+  FormContextType,
+  RJSFSchema,
+  StrictRJSFSchema,
+  WidgetProps,
+  UIOptionsType,
+} from '@rjsf/utils';
+
+import { MultiSelect, Select } from '@mantine/core';
+
+/**
+ * Returns and creates an array format required for mantine drop down
+ * @param {array} enumOptions- array of items for the dropdown
+ * @param {array} enumDisabled - array of enum option values to disable
+ * @returns {*}
+ */
+function createDefaultValueOptionsForDropDown<S extends StrictRJSFSchema = RJSFSchema>(
+  enumOptions?: EnumOptionsType<S>[],
+  enumDisabled?: UIOptionsType['enumDisabled']
+) {
+  const disabledOptions = enumDisabled || [];
+  if (!enumOptions) {
+    return [];
+  }
+  const options = enumOptions.map(({ label, value }, index) => ({
+    disabled: disabledOptions.indexOf(value) !== -1,
+    key: label,
+    label,
+    value: String(index),
+  }));
+  return options;
+}
+
+/** The `SelectWidget` is a widget for rendering dropdowns.
+ *  It is typically used with string properties constrained with enum options.
+ *
+ * @param props - The `WidgetProps` for this component
+ */
+export default function SelectWidget<T = any, S extends StrictRJSFSchema = RJSFSchema, F extends FormContextType = any>(
+  props: WidgetProps<T, S, F>
+) {
+  const {
+    id,
+    options,
+    label,
+    hideLabel,
+    required,
+    disabled,
+    readonly,
+    value,
+    multiple,
+    placeholder,
+    autofocus,
+    onChange,
+    onBlur,
+    onFocus,
+    rawErrors = [],
+  } = props;
+  const { enumDisabled, enumOptions } = options;
+
+  const emptyValue: string[] | string = multiple ? [] : '';
+  const dropdownOptions = createDefaultValueOptionsForDropDown<S>(enumOptions, enumDisabled);
+  const _onChange = (e: any) => onChange(enumOptionsValueForIndex<S>(e.value, enumOptions, emptyValue));
```

Comment:
@pkalisiewicz 

here now, you using `e.value`, yet the mantine `<Select />`'s props `onChange` will pass `string` and `<MaltiSelect />` will pass `string[]`

So, you shoule rewrite such as

```typescript
(e: string | string []) =>  ...
```

> please check [here : mantine Select](https://v5.mantine.dev/core/select/?t=props)
> and [here : mantie MultiSelect](https://v5.mantine.dev/core/multi-select/)